scripts/plugins/deploy_plugins.py

```diff
@@ -166,11 +168,9 @@ def Deploy_Plugins():
                     )
 
             except FileNotFoundError as e:
-                logger.warning(
-                    "No plugin file was found at path: [{}]".format(
-                        plugin_configuration_path
-                    )
-                )
+                msg, exit_code = Get_Doc("missing_file")
```

Comment:
```suggestion
                msg, _ = Get_Doc("missing_file")
```

Do we really need an `exit_code` here, considering it's not used below?
From what I understand the error here is not critical, right?

Sounds like the message here is also a bit off as the file is not absolutely required, but recommended:
```
Required file was not found. To fix this please add the following file:
```